css/src/admin-global.scss

```diff
@@ -311,8 +311,6 @@ td.column-wpseo-linked {
 		vertical-align: top;
 		/* inherits font size 14px */
 		line-height: 1.85714285; /* 26px vs. 28px from core */
-		/* Only necessary for IE11 */
-		min-height: 28px; /* 28px vs. 30px from core */
```

Comment:
Sorry for the misleading inline comment (it was just copied and pasted from WP core).
This needs to be temporarily restored to make _our_ input fields have the same height until core fixes it. See https://github.com/Yoast/wordpress-seo/pull/13803. On it.